config/sync/user.role.editor.yml

```diff
@@ -30,6 +32,7 @@ permissions:
   - 'create article content'
   - 'create campaign content'
   - 'create image media'
+  - 'create terms in categories'
```

Comment:
I'm unsure which roles are given to who. But should regular editors be able to create these? I thought it was only for admins / "webmaster".

Reply:
I understand. We have a "lesser" role called Mediator which I intentionally did not add this permission to.